lib/trino-parquet/src/main/java/io/trino/parquet/writer/ParquetWriter.java

```diff
@@ -93,10 +93,15 @@ public ParquetWriter(
 
     private static ParquetProperties getParquetProperties(ParquetWriterOptions writerOption)
     {
-        return ParquetProperties.builder()
+        ParquetProperties.Builder parquetPropertiesBuilder = ParquetProperties.builder()
```

Comment:
Using a name such as `builder` is more concise and does not lose clarity in this case.